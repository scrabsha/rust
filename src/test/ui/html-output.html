<pre style="margin:0px"><span style="color: red;font-weight:bold">error[E0580]</span><span style="font-weight:bold">: `main` function has wrong type</span>
<span style="">  </span><span style="color: blue;font-weight:bold">--&gt; </span><span style="">/home/ssha/iomentum/rust/src/test/ui/html-output.rs:10:1</span>
<span style="">   </span><span style="color: blue;font-weight:bold">|</span>
<span style="color: blue;font-weight:bold">10</span><span style=""> </span><span style="color: blue;font-weight:bold">| </span><span style="color: red;font-weight:bold">fn main(_: ())</span><span style=""> {}</span>
<span style="">   </span><span style="color: blue;font-weight:bold">| </span><span style="color: red;font-weight:bold">^^^^^^^^^^^^^^</span><span style=""> </span><span style="color: red;font-weight:bold">incorrect number of function parameters</span>
</pre><pre style="margin:0px"><span style="">   </span><span style="color: blue;font-weight:bold">|</span>
</pre><pre style="margin:0px"><span style="">   </span><span style="color: blue;font-weight:bold">= </span><span style="font-weight:bold">note</span><span style="">: expected fn pointer `fn()`</span>
<span style="">              found fn pointer `fn(</span><span style="font-weight:bold">()</span><span style="">)`</span>
</pre>
<pre style="margin:0px"><span style="color: red;font-weight:bold">error</span><span style="font-weight:bold">: aborting due to previous error</span>
</pre>
<pre style="margin:0px"><span style="font-weight:bold">For more information about this error, try `rustc --explain E0580`.</span></pre>
