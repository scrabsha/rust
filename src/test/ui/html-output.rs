// Allows us to test that the compiler emits a correct html output.
//
// The error in this test allows us to test the following features:
//   - the output is colored as it is when displayed on the terminal,
//   - the margin between each <pre> element is explicitely removed,
//   - text is in bold where expected.

// html-output

fn main(_: ()) {}
//~^ ERROR `main` function has wrong type [E0580]
