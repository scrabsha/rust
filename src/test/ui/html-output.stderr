error[E0580]: `main` function has wrong type
  --> $DIR/html-output.rs:10:1
   |
LL | fn main(_: ()) {}
   | ^^^^^^^^^^^^^^ incorrect number of function parameters
   |
   = note: expected fn pointer `fn()`
              found fn pointer `fn(())`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0580`.
